mitodl/mitxonline
# Add Checkout page UI

#### Pre-Flight checklist


#### What are the relevant tickets?
#387 

#### What's this PR do?
Adds checkout UI. Discount UI is also present but feature-flagged out. 

#### How should this be manually tested?
1. Ensure a product for a course run is set up and test credentials are in place for CyberSource
2. Enable the checkout test UI and go to /ecommerce-test/checkout to establish a basket
3. Go to /cart to view the cart
4. Clicking Place Order should redirect you to CyberSource

There isn't a receipt page here as that's in a different issue. Set ENABLE_DISCOUNT_UI flag to true to enable the discount UI (which is just the UI with placeholder text).

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/945611/154565297-5ae8afac-f7c9-44f3-9d16-bd0b88aacedc.png)
![image](https://user-images.githubusercontent.com/945611/154565193-696e217f-e56b-4983-b51a-13a68748ef30.png)
